### ISSUE TITLE
Get rid of, or set colors of, small dots within vertices

### ISSUE THREAD

**Issue by TorkelE** [JuliaPlots/GraphRecipes.jl#188]
I am plotting a graph, but there are some weird squares in the middle of each vertex. This causes problems as the colour of these dots cannot be controlled, and for large these dots take up a lot of space, preventing me from colouring the graph properly. Here's an example:
```julia
using GraphRecipes, Graphs, Plots
n = 15;
lattice = Graphs.SimpleGraphs.grid([n,n])
xs = vcat(fill(1:n,n)...)
ys = vcat(map(i->fill(i,n), 1:n)...)
colors = [fill(:red,125);fill(:blue,100)];
graphplot(lattice;x=xs,y=ys,nodecolor=colors,nodeshape=:rect,nodesize=4.2,linealpha=0.0,size=(1000,1000))
```
![image](https://user-images.githubusercontent.com/18099310/195879233-6e8e4fa6-45e7-475c-83ae-a38f76efcb9a.png)

here all of the dots in the middle are blue. In this case the overall picture is the same, but here I have an example where the entire plot becomes a single colour:
![image](https://user-images.githubusercontent.com/18099310/195879307-e6e4a628-98c6-424d-97e8-730946b5951e.png)
(The yellow squares fill up almost the entire nodes)

Is there a way to (preferably) remove these dots, or at least designate them colours identical to the underlying vertex?

**Comment by t-bltg** [comment]
Can you try https://github.com/JuliaPlots/GraphRecipes.jl/issues/154#issuecomment-1081248382 ?

Maybe related to https://github.com/JuliaPlots/GraphRecipes.jl/issues/175 or https://github.com/JuliaPlots/GraphRecipes.jl/issues/154.

**Comment by TorkelE** [comment]
Some progress, using `markerstrokewidth=0` does eliminate the black border around the central square, but the square remains:
![image](https://user-images.githubusercontent.com/18099310/195881321-f92ba37f-41e1-46ef-bfdc-aec9b9f6b79f.png)

**Comment by t-bltg** [comment]
Try to play with https://docs.juliaplots.org/stable/generated/attributes_series/.

A `PR` to set `markerstrokewidth=0`, and other potential fixes would be appreciated.

**Comment by TorkelE** [comment]
Sorry, tried to play around with what was there, but both the large and small things seem to be nodes in some way. What I have discovered is that while the big squares take the colors I provide, the small one always take the color of the last element in the `nodecolor` vector. I had a code at looking at the code, but didn't really get anywhere at all.

**Comment by gabrevaya** [comment]
Is there any advances on this issue?

**Comment by dsryu0822** [comment]
Still not resolved.

> Is there any advances on this issue?

No, it's still there.

**Comment by jsimons8** [comment]
Are there any updates on this issue? It's making the package unusable because my labels are barely readable now. :-(

**Comment by BeastyBlacksmith** [comment]
Should be solved by #199
